server/index.js

```diff
@@ -131,6 +132,37 @@ app.post('/api/cart', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.post('/api/orders', (req, res, next) => {
+
+  const { cartId } = req.session;
+
+  if (!(Number(cartId))) {
+    res.status(400).json({ err: 'There is no cartId' });
```

Comment:
Be sure to `return` here so that your endpoint doesn't continue trying to query the database.